appdev/guides/user-metrics.rst

```diff
@@ -77,17 +89,29 @@ User metrics are made up of two "formats"
 - metrics/messages for today (``format``)
 - metrics/messages for tomorrow (``emptyFormat``)
 
-The value of ``emptyFormat`` is what displays on the lock screen when no value has been stored in ``format``. In order to display a new value of ``format`` the metric must be updated with ``metric.update(x)`` (where x is the number of milliseconds until the update takes place).
+The value of ``emptyFormat`` is what displays on the lock screen when no value has been stored in ``format``. In order to display a new value of ``format`` the metric must be updated.
+
+Methods for the metric component (`source <https://daker.me/2013/11/adding-usermetrics-to-your-app-on-ubuntu-touch.html>`_):
+
+Set the metric to a specific amount:
+``metricID.update(x)`` (where x is a number of type `double` to set for a counter value).  The counter value can be included in the ``format`` setting by using ``%1``. ``metricID`` is the id specified in the Metric item.
+
+e.g. ``format: "%1 buttons pressed today"``
+
+Increment the metric:
+``metricID.increment(x)`` (where x is the amount to add to the current counter)
+
```

Comment:
This whole new section reads a bit uhm ... clumsily (is that a word) ... every new paragraph you make (two new lines) is rendered as it's own paragraph. Including the line starting with "e.g.". I think you could simply remove all new lines, thus putting all of your text into one paragraph. It will be clear to the reader.